src/components/dialog/dialog.tsx

```diff
@@ -1,16 +1,14 @@
 import { DialogHeading, ClosingActions } from './dialog.types';
-import { MDCDialog } from '@material/dialog';
```

Comment:
I found an edge case which will force us to fix things later, as soon as we have merged this PR.

Lime consultants have put many dialogs inside dialogs unfortunately. So basically, you can open a dialog on top of an existing dialog, by pressing a trigger element which is inside the first dialog. This has traditionally worked (with a buggy rendering in Chromium: https://github.com/Lundalogik/lime-elements/issues/2893).

to fake/test this behavior, paste this code in one of the example files:
```tsx
import { Component, h, State } from '@stencil/core';
@Component({
    tag: 'limel-example-dialog',
    shadow: true,
})
export class DialogExample {
    @State()
    private isOpen = false;
    public render() {
        return [
            <limel-button
                primary={true}
                label="Open"
                onClick={this.openDialog}
            />,
            <limel-dialog open={this.isOpen} onClose={this.closeDialog}>
                <p>This is a simple alert-dialog.</p>
                <limel-dialog open={this.isOpen} onClose={this.closeDialog}>
                    <p>This is a dialog, inside another dialog!</p>
                    <limel-button
                        label="Oh no..."
                        onClick={this.closeDialog}
                        slot="button"
                    />
                </limel-dialog>
                <limel-button
                    label="Ok"
                    onClick={this.closeDialog}
                    slot="button"
                />
            </limel-dialog>,
        ];
    }
    private openDialog = () => {
        this.isOpen = true;
    };
    private closeDialog = () => {
        this.isOpen = false;
    };
}
``` 

However, the problem now in this PR is that, as soon as you click the first trigger, it will open the nested dialog too, and the result will be like this:

<img width="619" alt="image" src="https://github.com/user-attachments/assets/a9b45fb3-e32d-48f7-8089-b1f625dd3d0c">


Previously, those buggy dialogs would still be usable if users used Firefox or Safari. However, with this PR, we will break their nested dialogs, if it's merged in the current state.

Reply:
> However, the problem now in this PR is that, as soon as you click the first trigger, it will open the nested dialog too, and the result will be like this:

That depends on how you have implemented your code :smile: In your example you are using `isOpen` to control both dialogs. If you have multiple dialogs you need a separate state for each one. Nothing in this PR changes that-

> Previously, those buggy dialogs would still be usable if users used Firefox or Safari. However, with this PR, we will break their nested dialogs, if it's merged in the current state.

How will they break?

Reply:
Oh silly me. I just copy/pasted some example code which I did for visual testing in the other issue like 10 years ago: https://github.com/Lundalogik/lime-elements/issues/2893 assuming that it's fine 🙈 😅 

Anyhow, it seems to be working fine in this PR now.